### PR TITLE
Clean up CloudWatch group UI

### PR DESCRIPTION
**Pull request by hunterwerlla**
- Breadcrumbs for where we are, later this will have actions to open group from string
- Remove search bar and replace with `SpeedSearch`
- Put refresh in the newly added idiomatic `Toolbar`. This will have an export action added to it later
- Fix sorting of the log streams + re-enable user sort + fix the width of last log stream time

## Screenshots (if appropriate)
Kapture 2020-03-20 at 12 51 49

## License
I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Comment by abrooksv** [comment]
Moving the toolbar to be same height as the table was the right call. Looks good

File: jetbrains-core/src/software/aws/toolkits/jetbrains/services/cloudwatch/logs/editor/CloudWatchLogGroup.kt [hunk 1]
```diff
@@ -63,46 +59,34 @@ class CloudWatchLogGroup(
...
+        tablePanel.setContent(scroll)
     }
 
     init {
-        locationInformation.text = "${project.activeCredentialProvider().displayName} => ${project.activeRegion().displayName} => $logGroup"
-        filterField.emptyText.text = message("cloudwatch.logs.filter_log_streams")
-        filterField.document.addDocumentListener(buildStreamSearchListener(groupTable))
+        val locationCrumbs = LocationCrumbs(project, logGroup)
+        locationInformation.crumbs = locationCrumbs.crumbs
+        locationInformation.border = JBUI.Borders.customLine(JBColor.border(), 0, 0, 1, 0)
```
Comment (abrooksv) [inline jetbrains-core/src/software/aws/toolkits/jetbrains/services/cloudwatch/logs/editor/CloudWatchLogGroup.kt]: Does this look the same?
```
IdeBorderFactory.createBorder(SideBorder.BOTTOM)
```

Comment (hunterwerlla) [reply jetbrains-core/src/software/aws/toolkits/jetbrains/services/cloudwatch/logs/editor/CloudWatchLogGroup.kt]: Yep, changed

File: jetbrains-core/src/software/aws/toolkits/jetbrains/services/cloudwatch/logs/editor/TableUtils.kt [hunk 2]
```diff
@@ -23,19 +24,18 @@ class LogStreamsStreamColumn : ColumnInfo<LogStream, String>(message("cloudwatch
...
-        return DateFormatUtil.getDateTimeFormat().format(timestamp)
-    }
+    private val renderer = ResizingDateColumnRenderer()
+    override fun valueOf(item: LogStream?): String? = item?.lastEventTimestamp()?.toString()
 
     override fun isCellEditable(item: LogStream?): Boolean = false
+    override fun getRenderer(item: LogStream?): TableCellRenderer? = renderer
 }
 
 class LogGroupTableSorter(model: ListTableModel<LogStream>) : TableRowSorter<ListTableModel<LogStream>>(model) {
```
Comment (abrooksv) [inline jetbrains-core/src/software/aws/toolkits/jetbrains/services/cloudwatch/logs/editor/TableUtils.kt]: What is behavior around scrolling if we change the sort and load more data?

Comment (hunterwerlla) [reply jetbrains-core/src/software/aws/toolkits/jetbrains/services/cloudwatch/logs/editor/TableUtils.kt]: This table can't currently load more data: it loads in all of the log groups. In a later PR the table needs to be extended to do that, but for right now I have no idea what would happen.

Comment (abrooksv) [reply jetbrains-core/src/software/aws/toolkits/jetbrains/services/cloudwatch/logs/editor/TableUtils.kt]: That makes sense...I'm getting my PRs confused

**Pull request merged**